qgreenland/util/config/compile.py

```diff
@@ -73,4 +73,7 @@ def compile_cfg(
             layer_tree=compiled_layer_tree,
         )
     except Exception as e:
-        raise exc.QgrConfigCompileError(f"Failed to compile config. {e}") from e
+        raise exc.QgrConfigCompileError(
+            f"Failed to compile config at '{config_dir}'. NOTE: Follow the traceback up"
+            f" to find the file at fault.\n{e}"
```

Comment:
I struggled for too long with the Pydantic error messages, which don't point to a file or line number. If you look really far up the traceback, you can find it.